Spotkanie3_Zadania/BMI/BMI/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿/*Zadanie 2 - Aplikacja konsolowa do przeliczania BMI:
+Stwórz nową solucję oraz projekty, które uważasz, że będą potrzebne
+Wymagania aplikacji: 
+użytkownik podaje wagę
+użytkownik podaje wzrost
+skorzystaj ze wzoru do obliczania BMI
+
+BMI = waga / (wzrost)^2
+
+przyjmij zakres : 18,5 – 24,9  jako prawidłowy, a zakres poniżej jako niedowagę, a zakres powyżej jako nadwagę
+wyświetl informacje dla użytkownika*/
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        Console.WriteLine("Welcome to BMI Calculator");
+        Console.WriteLine("We will now calculate your BMI");
+        Console.WriteLine("-------------------------");
+        Console.Write("Dear user, write me your name: ");
+        string name = Console.ReadLine();
+        while (true)
+        {
+            Console.Write($"Dear {name}, give me your weight: ");
+            int userWeight = int.Parse(Console.ReadLine());
+            Console.Write($"Dear {name}, tell me your height(in meters): ");
+            double userHeight = double.Parse(Console.ReadLine());
+            double bmi = CalculatorBMI(userWeight, userHeight);
+            Console.WriteLine($"Dear {name}, Your BMI is {Math.Round(bmi, 2)}. You have {Result(bmi)}");
+            Console.Write($"{name} do you want to continue[y/n]: ");
+            if (Console.ReadLine() == "y")
```

Comment:
Zastanow sie nad warunkami tu i w else, przetestuj

Reply:
dodałem tylko ToLower() ale pewnie nie o to chodzi.